kebechet/managers/update.py

```diff
@@ -592,12 +615,17 @@ def run(self, labels: list) -> typing.Optional[dict]:
             if os.path.isfile('Pipfile'):
                 _LOGGER.info("Using Pipfile for dependency management")
                 close_no_management_issue()
-                result = self._do_update(labels, pipenv_used=True)
+                result = self._do_update(labels, pipenv_used=True, req_dev=False)
             elif os.path.isfile('requirements.in'):
                 self._create_pipenv_environment()
                 _LOGGER.info("Using requirements.in for dependency management")
                 close_no_management_issue()
-                result = self._do_update(labels, pipenv_used=False)
+                result = self._do_update(labels, pipenv_used=False, req_dev=False)
+            elif os.path.isfile('requirements-dev.in'):
```

Comment:
Most likely people will have `requirements-dev.in` in the repo for development deps besides `requirements.in`. So it will be a good idea to move the `requirements-dev.in` part after `requirements.in` file handling.